include/tvm/tir/schedule/state.h

```diff
@@ -142,6 +142,8 @@ class ScheduleStateNode : public Object {
   /******** Property of blocks ********/
   /*! \brief Returns the BlockInfo correpsonding to the block sref */
   TVM_DLL BlockInfo GetBlockInfo(const StmtSRef& block_sref) const;
+  /*! \brief Recalculate the BlockInfo recursively under stmt */
+  TVM_DLL void UpdateBlockInfo(const Stmt& stmt);
```

Comment:
Are we updating the BlockInfo in the entire subtree? If so, let's find a better name for this method. Candidate: UpdateSubtreeBlockInfo